client/product-details/index.js

```diff
@@ -1,6 +1,54 @@
+/* global jQuery */
+// global wcpayStripeSiteMessaging
+
 /**
  * Internal dependencies
  */
-import bnplSiteMessaging from './bnpl-site-messaging';
+import { initializeBnplSiteMessaging } from './bnpl-site-messaging';
+
+jQuery( function ( $ ) {
+	const bnplPaymentMessageElement = initializeBnplSiteMessaging();
+	const { productVariations } = window.wcpayStripeSiteMessaging;
+	let { productId } = window.wcpayStripeSiteMessaging;
+
+	const resetBnplPaymentMessage = () => {
+		const quantity = $( '.quantity input' ).val();
+		productId = 'base_product';
+		bnplPaymentMessageElement.update( {
+			amount:
```

Comment:
We should also be supplying the currency.

Reply:
Added! thanks. 
ef11e5a